tests/src/test/scala/whisk/core/cli/test/ApiGwTests.scala

```diff
@@ -1212,4 +1212,29 @@ class ApiGwTests
         var rr = apiDelete(basepathOrApiName = nonexistentApi, expectedExitCode = ANY_ERROR_EXIT)
         rr.stderr should include (s"API '${nonexistentApi}' does not exist")
     }
+
+    it should "successfully list an API whose endpoints are not mapped to actions" in {
+        val testName = "CLI_APIGWTEST23"
+        var testapiname = "A descriptive name"
```

Comment:
I believe Scala naming conventions call for the use camel casing for var and val.